packages/design-system/src/components/web-components/ds-usa-banner.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { define } from './preactement/define';
+import { UsaBanner, UsaBannerProps } from '../UsaBanner/index';
+
+const attributes = ['class-name', 'root-id'];
```

Comment:
i don't think `class-name` is actually needed. looking at the react component, this only seems to exist to give teams a place to style the wrapper element. with web components, they can achieve the same thing by applying `class` to the root element. 

`root-id` makes more sense since the needed `id` is buried deeper within the component.

Reply:
I guess I've been thinking of this as a drop-in replacement for the react components, but this is actually a new API altogether. The fact that the `className` mapped to the `.ds-c-alert` element in the React component doesn't mean it needs to in this version. All of the special `--[state]` classes we have on `ds-c-alert*` are covered by the other attributes (like `variation` and `hide-icon`), so it's not like they need to be able to apply those themselves. I now realize we're not talking about Alert, but I guess the same principle applies.

Reply:
Anyway, yeah, I like the idea of keeping it simpler, especially if this component doesn't require a lot of customization usually

Reply:
It'll mean getting rid of a couple of your unit tests too, btw